copy-mdn.js

```diff
@@ -221,3 +221,38 @@ writeFileSync(
   JSON.stringify(printColorAdjustData)
 )
 
+const textDecorationColorData = bcdDataToCanIUseData(
+  bcd.css.properties['text-decoration-color'].__compat,
+  'text-decoration-color property'
+)
+writeFileSync(
+  './node_modules/caniuse-db/features-json/css-text-decoration-color.json',
+  JSON.stringify(textDecorationColorData)
+)
+
+const textDecorationLineData = bcdDataToCanIUseData(
+  bcd.css.properties['text-decoration-line'].__compat,
+  'text-decoration-line property'
+)
+writeFileSync(
+  './node_modules/caniuse-db/features-json/css-text-decoration-line.json',
+  JSON.stringify(textDecorationLineData)
+)
+
+const textDecorationStyleData = bcdDataToCanIUseData(
+  bcd.css.properties['text-decoration-style'].__compat,
+  'text-decoration-style property'
+)
+writeFileSync(
+  './node_modules/caniuse-db/features-json/css-text-decoration-style.json',
+  JSON.stringify(textDecorationStyleData)
+)
+
+const textDecorationShorthandData = bcdDataToCanIUseData(
+  bcd.css.properties['text-decoration'].shorthand.__compat,
+  'text-decoration shorthand property'
+)
+writeFileSync(
+  './node_modules/caniuse-db/features-json/css-text-decoration_shorthand.json',
```

Comment:
@ai Is the underscore ok here?
I followed the convention from caniuse (`mdn-css_properties_text-decoration_shorthand`)

Reply:
I used just `mdn-` prefix. But maybe it is a good idea to copy all. https://github.com/browserslist/caniuse-lite/commit/afe96e5ce3ee271e5bf03324521f822771be26e1

What do you think?

Reply:
I've renamed to `mdn-text-decoration-shorthand.json` I think this fits better with `mdn-css-unicode-bidi-isolate-override.json`